src/data_info/services/uuids.clj

```diff
@@ -29,22 +29,6 @@
    (irods/with-jargon-exceptions [cm]
        (path-for-uuid cm user uuid))))
 
-(defn ^IPersistentMap path-stat-for-uuid
```

Comment:
This function is nearly the same as `uuid-stat`, but without the filtering and adding a `uuid` field... which nothing used, and at least one thing explicitly had to filter out as a result. I changed everything that used it to just use `uuid-stat`.